### DISCUSSION TITLE
"Send Draft" not working

### DISCUSSION THREAD

**Discussion by krangor** [buttondown/discussions#52] [General]
Per the title; I've tried a few different email addresses and checked spam/junk, but nothing seems to be coming through.

**Comment by krangor** [comment]
Update: finally came through, after about a 20-minute delay, so semi-false alarm!

**Comment by jmduke** [comment]
I think we just had a bit of a backlog! Sorry about that.

**Comment by JJneid** [comment]
@jmduke i am having the same problem... draft not sent, been almost one hour

**Reply by jmduke** [reply]
@JJneid mind trying again? We had a small blip.
